server/src/domain/album/album-response.dto.ts

```diff
@@ -37,15 +37,6 @@ export const mapAlbum = (entity: AlbumEntity, withAssets: boolean): AlbumRespons
   const hasSharedLink = entity.sharedLinks?.length > 0;
   const hasSharedUser = sharedUsers.length > 0;
 
-  let startDate = assets.at(0)?.fileCreatedAt || undefined;
-  let endDate = assets.at(-1)?.fileCreatedAt || undefined;
-  // Swap dates if start date is greater than end date.
-  if (startDate && endDate && startDate > endDate) {
-    const temp = startDate;
-    startDate = endDate;
-    endDate = temp;
-  }
```

Comment:
I believe you still need this logic.

Reply:
No more needed since we use virtual columns which use `MIN` and `MAX` and don't rely on the order of assets